__tests__/app.test.js

```diff
@@ -149,6 +149,78 @@ describe("GET 200: /api/articles", () =>{
   
 })
 
+describe("POST 201: /api/articles/:article_id/comments", ()=>{
+  test("sends a 201 status and returns the posted comment after successfully inserting comment", () =>{
+    const comment = {username: "butter_bridge", body:"this is a comment"}
+    
+    return request(app).post("/api/articles/4/comments").send(comment).expect(201).then((response) =>{
+      const comments = response.body
+      expect(comments[0]).toHaveProperty("comment_id", expect.any(Number));
+        expect(comments[0]).toHaveProperty("votes", expect.any(Number));
+        expect(comments[0]).toHaveProperty("author", expect.any(String));
+        expect(comments[0]).toHaveProperty("body", expect.any(String))
+        expect(comments[0]).toHaveProperty("created_at", expect.any(String));
+        expect(comments[0]).toHaveProperty("article_id", expect.any(Number))
+      expect(comments[0].body).toBe("this is a comment")
+      expect(comments[0].author).toBe("butter_bridge")
+
+      return request(app).get("/api/articles/4/comments")
```

Comment:
This is one way to check the comments, but it is not necessary. You have already tested for this just above when checking the response that comes back